wiki/en/Running-a-Server.md

```diff
@@ -50,55 +49,47 @@ Unless you plan on hosting more than about 5 players on a slower-speed home conn
 
 - Running a server may require you to adjust any firewalls running on or outside of your machine or cloud host.
 
-- Running a Private Server at home will require you to [port forward](#running-a-private-server) on your router. When running a Public Server, port forwarding should not be necessary in most cases, but it's advisable to do so because some networks may not work properly with Jamulus in its default mode.
+- Running an Uregistered Server at home will require you to [port forward](#running-an-unregistered-server) on your router. When running a Registered Server, port forwarding should not be necessary in most cases, but it's advisable to do so because some networks may not work properly with Jamulus in its default mode.
 
 - Jamulus only has limited IPv6 support which needs to be enabled with a command line argument on the client and server. There are plans to expand IPv6 support.
 
-
 # Server Types
 
-You can run your server in one of three "modes" (either at home or on a 3rd party host):
+You can run your server in different ways (either at home or on a 3rd party host):
 
-### 1. Public
+### 1. Registered 
 
-Use this mode when you want anyone to join your server, whoever they are.   
-
-Your server will be listed in the Directory that clients use by default. Musicians can then discover and connect to your server.
+Your server will be listed in a Directory. By default, Jamulus has a list of Public Directories that clients can connect to. If you register with one of these, anyone can then discover and connect to your server. 
```

Comment:
```suggestion
Your server will be listed in a Directory. By default, Jamulus has a list of Public Directories that clients can connect to. If you register with one of these, your server is believed to be public and anyone can then discover and connect to your server. 
```

Not happy with my change, but we should stress the "public" part here.

Reply:
This might sort of clash with the FAQ entry on this, which says that some servers on the public Directories are sort of private.

Reply:
> In general, if somebody sets up a public server, they accept that anyone can play on it. Jamulus has no password protection or other authentication mechanisms. However, some servers may state their policies in the welcome message you will see in the chat window.

Implies that mostly all servers registered at a public directory are believed to be freely available

Reply:
> your server is believed to be public and

You can set up a firewall rule that lets you register your Server with Any Genre 1 but (unless `--showallservers` is used) won't be listed in the Connect window.  So adding any extras like this only confuses matters.

I agree _in principle_ that anyone using the Public Directories _should_ be registering their server for public use -- but we don't require it.